### DISCUSSION TITLE
V8Flags constructor is now private

### DISCUSSION THREAD

**Discussion by alphatheory-rlazarski** [caoccao/Javet#187] [General]
Hello, thanks for the great project. 

In 1.17, V8Flags changed. After reading the source code and unit tests, I cannot find a way to set the flags to custom values outside of the interop.options package. 

I just need to refactor this code to the latest javet but it's not clear to me how to get an instance of V8Flags to do it: 

                        V8Host v8Host = V8Host.getV8Instance();
                        v8Host.enableGCNotification();
                        V8Flags flags = v8Host.getFlags();
                        if (!flags.isSealed()) {
                                flags.setInitialHeapSize(2048);
                                flags.setMaxHeapSize(2048);
                                flags.setMaxOldSpaceSize(2048);
                        }

**Comment by caoccao** [comment]
V8 v10.5 introduces a breaking change. Please check out this [file](https://github.com/caoccao/Javet/blob/main/src/test/java/com/caoccao/javet/BaseTestJavet.java) for the new way of setting the flags. My bad, the doc hasn't been updated yet. I'll correct it soon.

**Comment by alphatheory-rlazarski** [comment]
Thanks for the reply. I looked at the code you linked. 

I am still unsure about instantiation because that unit test is in the same package as V8Flags, while when I try to instantiate it in my packages via "new V8Flags()" I get the compile error below: 

V8Flags() is not public in V8Flags; cannot be accessed from outside package

**Reply by alphatheory-rlazarski** [reply]
Actually I figured it out - the for loop is how it works now. Got it!
